packages/mdc-segmented-button/_index.scss

```diff
@@ -20,4 +20,109 @@
 // THE SOFTWARE.
 //
 
-// TODO: Add styles.
+@use "@material/button";
+@use "@material/theme/variables" as theme-variables;
+
+@use "@material/feature-targeting/functions" as feature-targeting-functions;
+@use "@material/typography/mixins" as typography-mixins;
+
+// Style for when element is on
+.mdc-segmented-button__segment--selected {
+  @include mdc-button.ink-color(rgba(theme-variables.prop-value(primary), 1));
+  @include mdc-button.outline-color(
+    rgba(theme-variables.prop-value(primary), 1)
+  );
+  @include mdc-button.container-fill-color(
+    rgba(theme-variables.prop-value(primary), 0.08)
+  );
+
+  // Avoids double borders
+  + .mdc-segmented-button__segment {
+    border-left: hidden;
```

Comment:
All styles should be defined inside feature target. See https://github.com/material-components/material-components-web/tree/master/packages/mdc-feature-targeting